protos/orchestrator_service.proto

```diff
@@ -678,6 +694,12 @@ service TaskHubSidecarService {
 
     // clean entity storage
     rpc CleanEntityStorage(CleanEntityStorageRequest) returns (CleanEntityStorageResponse);
+
+    // Abandons a single work item
+    rpc AbandonTaskActivityWorkItemAsync(AbandonActivityTaskRequest) returns (AbandonActivityTaskResponse);
+
+    // Abandon an orchestration work item
+    rpc AbandonTaskOrchestratorWorkItemAsync(AbandonOrchestrationTaskRequest) returns (AbandonOrchestrationTaskResponse);
```

Comment:
I believe we also need APIs for entity work items

Reply:
Updated.